README.md

```diff
@@ -1 +1,17 @@
-# portfolio
\ No newline at end of file
+# portfolio
+
+> building a mobile portfolio framework to display the logo and the header page
+
+## Built With
+
+- hyper text mark up language (HTML) and cascading stylesheet (CSS)
+
+
+👤 Hanningtone Monda
+
+- GitHub: [HanningtoneM)
+- Twitter: [@Hanningtone@254)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] **[OPTIONAL]**

Although the previous reviewer missed pointing out this, do bear in mind that they're **MANDATORY REQUIREMENTS** for your project, Try updating them:
- Your readme is not descriptive enough. 
Please refer to the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) to add the missing headers and ensuring you fill them with the details relevant to you projects.